cli/command/formatter/container_test.go

```diff
@@ -357,7 +357,6 @@ func TestContainerContextWriteJSON(t *testing.T) {
 		t.Fatal(err)
 	}
 	for i, line := range strings.Split(strings.TrimSpace(out.String()), "\n") {
-		t.Logf("Output: line %d: %s", i, line)
```

Comment:
I think it might be valuable to use this as a message in the assertions.

All the assert/require functions accept additional arguments which are printed as an additional message if the test fails. This can help debugging tests where there are assertions in loops, because the line numbers and error messages don't always give you enough information.

It would like something like this:

```go
msg := fmt.Sprintf("Output: line %d: %s", i, line)
var m map[string]interface{}
err := json.Unmarshal([]byte(line), &m)
require.NoError(t, err, msg)
assert.Equal(t, expectedJSONs[i], m, msg)
```

The same applies to the other changes